### DISCUSSION TITLE
Missing js file when building custom keycloak version

### DISCUSSION THREAD

**Discussion by wolframite** [mabartos/keycloak-adaptive-authn#17] [Q&A]
I followed the instructions of how to build the custom keycloak version for the authn-policy module, but I found some problems here:
- It seems that there's no 24.0.4 tag in the keycloak repo, only 24.0.2
- In {keycloak-root}/js/libs/keycloak-admin-client/src/client.ts `organizations.js` is imported from the resources folder, but I can't find it anywhere in this repo or in the keycloak repo

The latter one breaks the build. Does anyone have a similar problem or an idea how to fix it? I'd prefer to build keycloak myself rather than relying on the pre-built artifacts.

**Comment by wolframite** [comment]
I found the files in this keycloak fork: https://github.com/keycloak/keycloak – I'll try to build it with this repo

**Comment by mabartos** [comment]
Seems not relevant anymore.
